src/store/mutations.js

```diff
@@ -217,4 +217,19 @@ export default {
 	removeMessage(state, {accountId, folderId, id}) {
 		Vue.delete(state.messages, normalizedMessageId(accountId, folderId, id))
 	},
+	changeFolderSubscription(state, {accountId, folderId, isSubscribed}) {
+		const uid = normalizedFolderId(accountId, folderId)
+		const folder = state.folders[uid]
+		var attributes = folder.attributes
```

Comment:
can be `const` because you mutate the variable only internally, but do not assign a new value